docs/UserGuide.md

```diff
@@ -15,11 +15,14 @@ _Say goodbye to chaos and hello to an organized and efficient classroom experien
 </p>
 
 
-F.A.K.E.J.A.R.V.I.S. is the **ultimate solution for CS1101S Avengers who manage a multitude of tutoring tasks.** This platform empowers you to efficiently track assignment gradings, monitor student participation, and seamlessly plan tutorials, consultations, and mastery checks. This is your one-stop solution to managing your time and students as a CS1101S Avenger!
+F.A.K.E.J.A.R.V.I.S. is your **friendly companion for efficient task and administrative management**, allowing you to excel as a CS1101S Avenger. With this tol, you can effortlessly track assignment gradings, monitor student participation, and seamlessly plan tutorials, consultations, and mastery checks. 
 
-The application is designed for efficient usage through a Command-Line Interface, all while preserving the advantages of having a Graphical User Interface.
+As Avengers ourselves, we understand the importance of managing both our time and our students effectively. F.A.K.E.J.A.R.V.I.S. is the one-stop solution to streamlining your workload, making your life easier. It is designed to empower you with the benefits of a Command-Line Interface(CLI), all while preserving the advantages of having a Graphical User Interface(GUI).
 
-If you're new to the Command-Line Interface and need assistance, this User Guide is here to help you become familiar with it and use the application with ease.
+If you're new to the CLI and need assistance, don't worry! This UserGuide is your trusted assistant, and will guide you through every step and ensure that you can unlock the full potential of F.A.K.E.J.A.R.V.I.S. Let's get started!
```

Comment:
If you're new to the CLI or you're simply unfamiliar with the commands that FAKEJARVIS offers, ....

This removes the fallacy that she preached about